CLI/models/attributes.go

```diff
@@ -191,3 +193,31 @@ func expandStrToVector(slot string) ([]string, error) {
 		}
 	}
 }
+
+// Check the format of a sizeU attribute and return it as a float64 is possible
+func CheckSizeUFormat(value any) (float64, error) {
```

Comment:
Utils has a IsNumeric function, could simplify these two here

Reply:
I'll use utils.GetFloat() instead : it does the same work as my functions..